.github/workflows/build.yaml

```diff
@@ -260,6 +260,21 @@ jobs:
               run: |
                   scripts/run_in_build_env.sh 'pip3 install ./out/controller/python/chip-0.0-cp37-abi3-linux_x86_64.whl'
                   scripts/run_in_build_env.sh '(cd src/controller/python/test/unit_tests/ && python3 -m unittest -v)'
+            - name: Build all clusters app for test
+              timeout-minutes: 50
+              run: |
+                  scripts/examples/gn_build_example.sh examples/all-clusters-app/linux out/all_clusters_debug chip_enable_wifi=false
+            - name: Run Python REPL API Tests
+              timeout-minutes: 5
+              run: |
+                  scripts/run_in_build_env.sh '
+                      rm -rf /tmp/chip*
+                      ./out/all_clusters_debug/chip-all-clusters-app &
```

Comment:
We should structure this in a such a way that it's easy to tell from the logs which are server-side vs. controller-side logs. The way it's written here, it's going to be difficult to tell apart.

Reply:
Added a wrapping script, now it will append `[APP][STDOUT]` to the server side log and `[TEST][STDOUT]` to the controller side logs, with a formatted timestamp in the log output.